mmv1/third_party/terraform/services/compute/compute_instance_helpers.go.erb

```diff
@@ -656,7 +692,7 @@ func schedulingHasChangeRequiringReboot(d *schema.ResourceData) bool {
 	oScheduling := o.([]interface{})[0].(map[string]interface{})
 	newScheduling := n.([]interface{})[0].(map[string]interface{})
 
-	return hasNodeAffinitiesChanged(oScheduling, newScheduling)
+	return hasNodeAffinitiesChanged(oScheduling, newScheduling) || hasMaxRunDurationChanged(oScheduling, newScheduling)
```

Comment:
This suggested change should mean that the changes only appear in the Beta provider

```suggestion
<% unless version == 'ga' -%>
	return hasNodeAffinitiesChanged(oScheduling, newScheduling) || hasMaxRunDurationChanged(oScheduling, newScheduling)
<% else -%>
	return hasNodeAffinitiesChanged(oScheduling, newScheduling)
<% end -%>
```